settings.gradle

```diff
@@ -117,7 +117,7 @@ include 'data-prepper-expression'
 include 'data-prepper-plugins:mutate-string-processors'
 include 'data-prepper-plugins:s3-source'
 include 'data-prepper-plugins:s3-sink'
-include 'data-prepper-plugins:rss-source'
+//include 'data-prepper-plugins:rss-source'
```

Comment:
Any reason to remove this in this PR?

Reply:
I should not have removed it, but the build has failed multiple times in RSS. I will add it back